src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -24,6 +24,12 @@ method java.lang.String substring int int
 method java.lang.String replace java.lang.CharSequence java.lang.CharSequence
 method java.util.Collection contains java.lang.Object
 method java.util.concurrent.Callable call
+method java.util.HashMap get java.lang.Object
+method java.util.HashMap put java.lang.Object java.lang.Object
+method java.util.LinkedHashMap get java.lang.Object
+method java.util.LinkedHashMap put java.lang.Object java.lang.Object
```

Comment:
These should be redundant given the whitelists of `Map` methods, since the interceptor will look up the location of the method definition, not override/implementation. See for example

    method java.util.List get int

above.